src/FailureStrategy/ClearMetaStrategy.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\FailureStrategy;
+
+use Yiisoft\Yii\Queue\Message\MessageInterface;
+
+final class ClearMetaStrategy implements FailureStrategyInterface
```

Comment:
Sounds like the name is describing internal stuff instead of behavior for end user.